x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -149,9 +149,15 @@ export interface ResponseActionGetFileParameters {
   path: string;
 }
 
+export interface ResponseActionsExecuteParameters {
+  command: string;
+  timeout: number;
```

Comment:
Timeout is optional

Reply:
oh yeah 👍

Reply:
b6f3c7e65c109142e16be1dfbfd6ea661291e4f7